modules/game/src/main/BinaryFormat.scala

```diff
@@ -112,6 +112,21 @@ object BinaryFormat {
       val l = (b1 << 24) + (b2 << 16) + (b3 << 8) + b4
       if (l == 0) 0 else l + decay
     }
+
+    private def writeClockLimit(limit: Int) = {
+      // The database expects a byte for a limit, and this is limit / 60.
+      // For 0.5+0, this does not give a round number, so there needs to be
+      // an alternative way to describe 0.5.
+      // The max limit where limit % 60 == 0, returns 180 for limit / 60
+      // So, for the limits where limit % 30 == 0, we can use the space
+      // from 181-255, where 181 represents 0.5 (and 182 would represent 1.5,
+      // but that doesn't exist).
+      if (limit % 60 == 0) limit / 60 else (limit - 30) / 60 + 181
+    }
+
+    private def readClockLimit(b: Int) = {
+      if (b < 181) b * 60 else (b - 181) * 60 + 30
+    }
```

Comment:
lol nice hack. I think it's acceptable. Though it closes the door on longer time controls, like 200 minutes.